src/Test/Integration/DbHelperAbstract.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace ShoppinPal\YapepCommon\Test\Integration;
+
+
+use Behat\Gherkin\Node\TableNode;
+use PHPUnit_Framework_Assert;
+use YapepBase\Database\DbConnection;
+
+abstract class DbHelperAbstract
+{
+
+    /**
+     * @return DbConnection
+     */
+    abstract public function getDbConnection();
+
+    /**
+     * @param string $tableName
+     * @param int    $expectedEntryCount
+     */
+    public function assertTableEntryCount($tableName, $expectedEntryCount)
+    {
+        $query
+            = '
+            SELECT
+                COUNT(*)
+            FROM
+                ' . $tableName . '
+        ';
+
+        $entryCount = $this->getDbConnection()->query($query)->fetchColumn();
+
+        PHPUnit_Framework_Assert::assertEquals($expectedEntryCount, $entryCount);
+    }
+
+    /**
+     * @param string    $tableName
+     * @param TableNode $table
+     */
+    public function assertTableContents($tableName, TableNode $table)
+    {
+        $expectedRows = $table->getHash();
+
+        $fields = array_keys($expectedRows[0]);
+
+        $query
+            = '
+            SELECT
+                ' . implode(', ', $fields) . '
+            FROM
+                ' . $tableName . '
+        ';
```

Comment:
You're right. I added an array as a third parameter to be able to set multiple fields for ordering